helpers/event/event_slack_notifier.py

```diff
@@ -0,0 +1,11 @@
+import requests
+import celery
+from config import Config
+
+"""
+This function will call slack bot url
+"""
+
+@celery.task(name="notify-slack")
+def notify(event_Id):
+    requests.get(url=Config.notify_url+str(event_Id))
```

Comment:
Why are you casting `event_Id` to a string. Is it necessary? Also, please refer to variable naming conventions for `event_Id`. (https://www.python.org/dev/peps/pep-0008/#function-and-variable-names)

Reply:
the `event_id` is an integer and the `URL` is a string
so, without converting `event_id` to a string was causing an `error` @joshuaocero